src/main/java/seedu/address/logic/commands/DeleteInterviewCommand.java

```diff
@@ -42,7 +45,9 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Interview interviewToDelete = lastShownList.get(targetIndex.getZeroBased());
-        model.deleteInterview(interviewToDelete);
+        Applicant applicantToUpdate = interviewToDelete.getInterviewApplicant();
+        model.setApplicant(applicantToUpdate, applicantToUpdate.getApplicantWithoutInterview());
```

Comment:
Maybe the set applicant can be put inside the deleteInterview method in ModelManager.